hathor/transaction/block.py

```diff
@@ -38,11 +38,11 @@
 settings = HathorSettings()
 cpu = get_cpu_profiler()
 
-# Version (H), outputs len (B)
-_FUNDS_FORMAT_STRING = '!HB'
+# Signal bits (B), version (B), outputs len (B)
+_FUNDS_FORMAT_STRING = '!BBB'
 
-# Version (H), inputs len (B) and outputs len (B)
-_SIGHASH_ALL_FORMAT_STRING = '!HBB'
+# Signal bits (B), version (B), inputs len (B) and outputs len (B)
+_SIGHASH_ALL_FORMAT_STRING = '!BBBB'
```

Comment:
This is not used. Can we remove it?

Reply:
I don't think we should do it in this PR.